umh-core/integration/integration_test.go

```diff
@@ -647,6 +626,78 @@ var _ = Describe("UMH Container Integration", Ordered, Label("integration"), fun
 			GinkgoWriter.Println("Benthos scaling test completed successfully")
 		})
 	})
+
+	Context("with redpanda enabled but no benthos services", Label("redpanda-only"), func() {
```

Comment:
maybe also add a test that checks under the given CPU and RAM limitations (2 cores, 3GB of memory) that 10 benthos services can write into kafka. with this, we can catch any regressions here

Reply:
Added :)